packages/expo-crypto/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 ### 💡 Others
 
+- Rewrite android code to Kotlin. ([#14425](https://github.com/expo/expo/pull/14425) by [@kkafar](https://github.com/kkafar))
```

Comment:
```suggestion
- Rewrite Android code to Kotlin. ([#14425](https://github.com/expo/expo/pull/14425) by [@kkafar](https://github.com/kkafar))
```
Be sensitive about the correct usage of the different names, like `iOS`, `Android`, `Kotlin` etc 😉
Same applies to any git/GitHub work (like PR name/description, commit tags/message, etc).